packages/common/README.md

```diff
@@ -104,12 +104,41 @@ files like `mainnet.json` in the `chains` directory, or to the `Chain` type in [
 
 ### Working with private/custom chains
 
-There are two ways to set up a common instance with parameters for a private/custom chain:
+There are two distinct APIs availale for setting up custom(ized) chains.
 
-1. You can pass a dictionary - conforming to the parameter format described above - with your custom values in
-   the constructor or the `setChain()` method for the `chain` parameter.
+### Activate with a single custom Chain setup
 
-2. You can base your custom chain's config in a standard one, using the `Common.forCustomChain` method.
+If you want to initialize a `Common` instance with a single custom chain which is then directly activated
+you can pass a dictionary - conforming to the parameter format described above - with your custom chain 
+values to the constructor using the `chain` parameter or the `setChain()` method, here is some example:
+
+```typescript
+import myCustomChain from './[PATH]/myCustomChain.json'
+const common = new Common({ chain: myCustomChain })
+```
+
+If you just want to change some certain parameters on a chain configuration it can also be conveniened to use 
+the `Common.forCustomChain()` method. With this method you can base your custom chain configuration with
+a standard one (so using all the values from `baseChain` as the default values) and then just provide the
+parameters you want to override:
+
+```typescript
+const customChainParams = { name: 'custom', chainId: 123, networkId: 678 }
+const customChainCommon = Common.forCustomChain('mainnet', customChainParams, 'byzantium')
+```
+
+### Initialize using customChains Array
+
+A second way for custom chain initialization is to use the `customChains` constructor option. This
+option comes with more flexibility and allows for an activation-independent initialization (e.g. to
+initialize with `mainnet` activated but with an additional `myCustomChain` available) as well as
+initializing with an arbitrary number of your own custom chain configuration objects:
```

Comment:
From this description I am not entirely sure what this does. I can initialize a common which is set to the default `mainnet`, but I can also pass an array with extra chain parameters - how do I activate them? Should I activate them by setting it as hardfork?

Reply:
Have rewritten this, hope this makes it more clear.